packages/engine/paima-runtime/tsconfig.json

```diff
@@ -11,7 +11,6 @@
     { "path": "../../node-sdk/paima-db" },
     { "path": "../paima-rest/tsconfig.build.json" },
     { "path": "../paima-sm" },
-    { "path": "../../node-sdk/paima-broker/tsconfig.build.json" },
```

Comment:
we need this. Please don't remove it

Reply:
is this warning a false positive then?

```
> @paima/root@0.0.0 lint:configs
> ts-node -P ./tsconfig.base.json ./tools/scripts/check-implicit-dependencies.ts

warn: package packages/engine/paima-runtime/tsconfig.json has unnecessary references
node-sdk/paima-broker
```

I removed it because of that and because it didn't seem to break anything, but I can undo the change. It would be nice to hide the warning if this is the case though.

Reply:
Yes, it's a false positive. It's an issue in `nx` itself afaik and the warning does have some implications about how nx handles things, so there is not much we can do other than ignore it